pkg/crc/machine/machine.go

```diff
@@ -378,6 +378,23 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		return *result, errors.Newf("Error starting kubelet: %s", err)
 	}
 	if kubeletStarted {
+		// With 4.3, when cluster come up and request header client-ca regenerated then
```

Comment:
"when the cluster comes up", this only happens on first start, I guess this should be explicit.
"request header client-ca regenerated " I'm not sure I understand this part of the sentence :-/